packages/data-explorer/src/components/TableSelect.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <b-nav class="bread-crumb-dropdown">
+    <b-nav-item-dropdown
+        :id="label"
+        :text="label"
+        toggle-class="bread-crumb-item-dropdown"
+        dropup
+        no-caret
+    >
+    <template v-slot:button-content>
+        <span class="bread-crumb-select">
+        <h1>
+            {{label}}
+        </h1>
+        <svg data-v-041edcbe=""
+            class="chevron mr-2 svg-inline--fa fa-caret-right fa-w-6"
+            aria-hidden="true"
+            focusable="false"
+            data-prefix="fas"
+            data-icon="caret-right"
+            role="img" xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 192 512"
+            style="transform: rotate(90deg); transition: transform 0.2s ease 0s;">
+            <path
+            data-v-041edcbe=""
+            fill="currentColor"
+            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
+            class="">
+            </path>
+        </svg>
+        </span>
+        </template>
+    <template v-for="table in packageTables">
+        <b-dropdown-item :key="table.id" :to="table.id">{{table.label}}</b-dropdown-item>
```

Comment:
Maybe add a close button to keep the style inline with the add/change filter box? I close boxes like this by clicking outside it. But having a close button might help other users, and makes it look complete.

Reply:
What happens if there are 100 entities to switch to?

Reply:
this is the bootstrap ui default, suggest to keep it like this for now

Reply:
> What happens if there are 100 entities to switch to?

a long list is shown